File: go-backend/model/xray_tls_cert.go
package model

type XrayTlsCert struct {
	ID          int64  `gorm:"primaryKey;autoIncrement" json:"id"`
	NodeId      int64  `gorm:"column:node_id" json:"nodeId"`
	Domain      string `gorm:"column:domain" json:"domain"`
	PublicKey   string `gorm:"column:public_key" json:"publicKey"`
	PrivateKey  string `gorm:"column:private_key" json:"privateKey"`
	AutoRenew   int    `gorm:"column:auto_renew" json:"autoRenew"`
	ExpireTime  *int64 `gorm:"column:expire_time" json:"expireTime"`
	CreatedTime int64  `gorm:"column:created_time" json:"createdTime"`
	UpdatedTime int64  `gorm:"column:updated_time" json:"updatedTime"`
}

func (XrayTlsCert) TableName() string {
	return "xray_tls_cert"
}

File: springboot-backend/src/main/java/com/admin/entity/XrayTlsCert.java
package com.admin.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.io.Serializable;

@Data
@TableName("xray_tls_cert")
public class XrayTlsCert implements Serializable {

    private static final long serialVersionUID = 1L;

    @TableId(value = "id", type = IdType.AUTO)
    private Long id;

    private Long nodeId;

    private String domain;

    private String publicKey;

    private String privateKey;

    private Integer autoRenew;

    private Long expireTime;

    private Long createdTime;

    private Long updatedTime;
}
